File: PRAK110-2510817310005-Tristan_Nathan_Naurell_Prasetyo.py
import math

alas, tinggi = 5, 12
sisiA = alas
sisiB = tinggi
sisiC = int(math.sqrt(alas*2 + tinggi*2))
keliling = sisiA + sisiB + sisiC
luas = (alas * tinggi) // 2

print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")

print("Jawab :")
print(f"Sisi A = {sisiA} cm")
print(f"Sisi B = {sisiB} cm")
print(f"Sisi C = {sisiC} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm")

File: PRAK110-2510817310005-Tristan_Nathan_Naurell_Prasetyo.c
#include <stdio.h>
#include <math.h>

int main() {
    int alas = 5;
    int tinggi = 12;
    int sisiA = alas;
    int sisiB = tinggi;
    int sisiC = sqrt((alas * alas) + (tinggi * tinggi));
    int keliling = sisiA + sisiB + sisiC;
    int luas = (alas * tinggi) / 2;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);

    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", sisiA);
    printf("Sisi B = %d cm\n", sisiB);
    printf("Sisi C = %d cm\n", sisiC);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", luas);

    return 0;
}